src/content/docs/aspen.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Aspen
+description: Check out Aspen, the privacy focused HTTP client.
+---
+
+Aspen is a Treblle's free, native macOS application designed specifically for testing REST APIs. Built with privacy in mind, it requires no login and stores no data, ensuring a smooth, local testing experience.
+
+Aspen comes with an integrated [AI assistant](https://treblle.com/product/alfred) to help automate API integration tasks, such as generating data models, Open API Specifications (OAS), and integration code.
+
+## Getting Started
+
+1. **Download Aspen**  
+    You can download Aspen from the [Apple Mac Store](https://apps.apple.com/us/app/aspen-your-apis-best-friend/id6475777033).
+
+   <img
+     src="/platform/aspen/download.png"
+     width="800"
+     style={{ borderRadius: "8px" }}
+   />
+
+2. **Install Aspen**  
+   After downloading, install the app on your Mac and open it.
+
+   <img
+     src="/platform/aspen/aspen.png"
+     width="800"
+     style={{ borderRadius: "8px" }}
+   />
+
+3. **Create a New Request**:  
+   In Aspen, create a new request by specifying the method (GET, POST, PUT, DELETE) and the URL of your API endpoint.
+
+   <img
+     src="/platform/aspen/example.png"
+     width="800"
+     style={{ borderRadius: "8px" }}
+   />
+
+- **Add Request Data**:  
+   For POST and PUT requests, provide the required data in the request body in JSON format.
+
+  <img
+    src="/platform/aspen/post.png"
+    width="800"
+    style={{ borderRadius: "8px" }}
+  />
+
+- **View Response**:  
+   Aspen displays the response from the API directly in the interface, showing the status code and returned data.\
+
+## Aspen Collections
+
+- Aspen's Collections feature allows you to organize your API requests into groups for easy management and reuse.
+
+- You can create folders and sub-folders within Collections to organize your requests by API flow, service, or any other criteria.
+
+#### Create a Collection:
+
+- From the History sidebar, save your API requests as a Collection and name them according to your needs.
+
+  <img
+    src="/platform/aspen/collection.png"
+    width="800"
+    style={{ borderRadius: "8px" }}
+  />
+
+#### Search Through Collections:
+
+- You can search through your saved Collections to quickly find specific requests. This is especially helpful when managing a large number of requests.
+
+  <img
+    src="/platform/aspen/search.png"
```

Comment:
All other images are clear and readable. This image isn't. It is not clear what is happening in it. I would zoom it and take another.

Reply:
Zoomed and updated.